### PR TITLE
[7]Develop A customer form page using HTML

### PR DESCRIPTION
**Pull request by Fiyin985**
## Task: `I made the form pages and linked them`
-

### CODE REVIEW

File: submissions/fiyingabriel/htmlForms/signUp.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   <label>Language:</label>
+   <input list="country-list" name="country" id="country1">
+
+   <datalist id="country-list">
+   	    <option value="English"/>
+   	    <option value="Spanish"/>
+   	    <option value="French"/>
+   	    <option value="Mandarin"/>
+   </datalist>  <br><br>
+   <div align="center"><h3>Already Have An Account?</h3><a href="logIn.html">Log In</a><h3>Back to Home Page</h3><a href="htmlTask.html">Click Here!</a></div>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/signUp.html:31]: Remove the styling here `align`.

File: submissions/fiyingabriel/htmlForms/logIn.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+   <label for="email"> Email:</label>
+   <input type="email" name="email" id="email" placeholder="Your Email Address"> <br><br>
+
+
+
+   <label>Password</label>
+   <input type="password" name="password"/><br><br>
+   
+   </datalist>  <br><br>
+   <div align="center"><h3>Dont Have An Account?</h3><a href="signUp.html">SignUp</a><h3>Back to Home Page</h3><a href="htmlTask.html">Click Here!</a></div>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/logIn.html:23]: Remove the styling here `align`.

File: submissions/fiyingabriel/htmlForms/logIn.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="styles.css">
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/logIn.html:5]: Remove the link to css here.

File: submissions/fiyingabriel/htmlForms/logIn.html (hunk 2 above)
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/logIn.html:23]: Remove `align` here

File: submissions/fiyingabriel/htmlForms/logIn.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+	<h1>Login</h1><br><br><br><br><br>
+	<form action="#" method="post" enctype="multiort/form-data">
+   <fieldset>
+   	     <label for="firstname">User Name</label>
+   <input type="text" name="firstname" id="firstname"/> <br><br>
+  
+   <label for="email"> Email:</label>
+   <input type="email" name="email" id="email" placeholder="Your Email Address"> <br><br>
+
+
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/logIn.html:17]: Remove the extra white space here.

File: submissions/fiyingabriel/htmlForms/signUp.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="styles.css">
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/signUp.html:5]: Remove  the link to css here

File: submissions/fiyingabriel/htmlForms/htmlTask.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE HTML>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/htmlTask.html:1]: Remove this file. Htmlform should only contain forms.

File: submissions/fiyingabriel/htmlForms/contactUs.html [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/contactUs.html:1]: Remove the images in htmlforms folder. @Fiyin985

File: submissions/fiyingabriel/htmlForms/contactUs.html [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+    <h2>Subscribe to get the latest info on our products</h2>
+     <form action="#" method="post" enctype="multiort/form-data">
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/contactUs.html:9]: Remove the attributes `action`, `method`, `post` and `enctype` from the form tag

File: submissions/fiyingabriel/htmlForms/contactUs.html [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+         <label for="email"> Email:</label>
+         <input type="email" name="email" id="email" placeholder="Your Email Address"> <br><br>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/contactUs.html:11]: Your contact form only has email as the input field,  no message input field???

File: submissions/fiyingabriel/htmlForms/htmlTask.html [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+</head>
+  
+<body>
+	<a href="logIn.html">Login</a><br><br>
+	<a href="signUp.html">Sign Up</a><br><br>
+	<a href="contactUs.html">News</a>
+	<h1>Apple iPhone X - 64GB</h1>
+    <div>
+       <p><img src="149543_1549727850.jpg"></p>
+       <h1 align="left">N250,000</h1>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/htmlTask.html:18]: Remove the styling here `align`.

File: submissions/fiyingabriel/htmlForms/htmlTask.html [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+       <button type="submit">Buy Now</button>   
+    </div>
+	<h2>Descriptions</h2>
+	<table border="2">
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/htmlTask.html:22]: Remove the styling here `border`.

File: submissions/fiyingabriel/htmlForms/logIn.html (hunk 2 above)
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/logIn.html:23]: Remove the styling here `align`.

**Comment by Fiyin985** [comment]
Ive updated the file

**Comment by Fiyin985** [comment]
how do i do [8] ? it is the same file as [7] im using

File: submissions/fiyingabriel/htmlForms/signUp.html [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+	<h1>Sign Up</h1><br><br><br><br><br>
+	<form>
+   <fieldset>
+   	     <label for="firstname">User Name</label>
+   <input type="text" name="firstname" id="firstname"/> <br><br>
+  
+   <label for="email"> Email:</label>
+   <input type="email" name="email" id="email" placeholder="Your Email Address"> <br><br>
+
+
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/signUp.html:16]: Remove the extra line space here @Fiyin985

**Comment by ToluAkin** [comment]
> how do i do [8] ? it is the same file as [7] im using

You would have to create a separate pull request for [8] @Fiyin985

**Comment by Fiyin985** [comment]
Ive updated it

**Comment by Fiyin985** [comment]
i tried to make a new pull request but i cant

**Comment by ToluAkin** [comment]
> i tried to make a new pull request but i cant

But the images are still there as well as the other product page you created @Fiyin985

**Pull request closed without merging**